x-pack/plugins/reporting/server/export_types/csv/create_job.ts

```diff
@@ -20,6 +20,9 @@ export const createJobFnFactory: CreateJobFnFactory<
   const crypto = cryptoFactory(config.get('encryptionKey'));
 
   return async function createJob(jobParams, context, request) {
+    logger.warn(
+      `The "/generate/csv" endpoint is deprecated and will be removed in Kibana 8.0. Please recreate the POST URL used to automate this CSV export.`
```

Comment:
Suggestion: perhaps we could rephrase this as:

```
This CSV export is deprecated and will be removed in Kibana 8.0. Please recreate the POST URL to continue generating this CSV export. See <docs link with screenshots>.
```

My concern with the current version is that mentioning an endpoint `/generate/csv` is a bit too implementation-level detail. WDYT?

Additionally, I think we should wrap this in `i18n.translate`.

If we can link them to somewhere in the docs that would be A+, but this can definitely be addressed in a follow up PR if you agree!